.github/workflows/testing-and-deployment.yml

```diff
@@ -150,9 +150,6 @@ jobs:
       matrix:
         include:
           # numeric numpy versions are ~= conditions, e.g. "1.23" means "numpy~=1.23.0"
-          - python-version: "3.8"
-            vtk-version: "9.0.3" # Requires numpy~=1.23
```

Comment:
Are we also dropping support for this vtk version?  It looks like can be installed with putting  Python 3.9, so we should probably keep it in the matrix somehow. Either add a second Python 3.9 or reorder the vtk-versions in the matrix.

Reply:
We really ought to continue testing VTK 9.0.3. Good call on having two Py3.9 entries for VTK 9.0.3 and VTK 9.1

Reply:
Now this is resolved thanks all for your help!